Index: src/Controller/OffreStagesController.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?php\r\n\r\nnamespace App\\Controller;\r\n\r\nuse Symfony\\Bundle\\FrameworkBundle\\Controller\\AbstractController;\r\nuse Symfony\\Component\\HttpFoundation\\Response;\r\nuse Symfony\\Component\\Routing\\Annotation\\Route;\r\n\r\nclass OffreStagesController extends AbstractController\r\n{\r\n    #[Route('/offre/stages', name: 'app_offre_stages')]\r\n    public function index(): Response\r\n    {\r\n        return $this->render('frontOffice/offre_stages/recrutement.html.twig', [\r\n            'controller_name' => 'OffreStagesController',\r\n        ]);\r\n    }\r\n    \r\n    \r\n    \r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Controller/OffreStagesController.php b/src/Controller/OffreStagesController.php
--- a/src/Controller/OffreStagesController.php	
+++ b/src/Controller/OffreStagesController.php	
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use App\Repository\OffreStageRepository;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
@@ -9,12 +10,24 @@
 class OffreStagesController extends AbstractController
 {
     #[Route('/offre/stages', name: 'app_offre_stages')]
-    public function index(): Response
+    public function index( OffreStageRepository $offreStageRepository): Response
     {
+        $liste = $offreStageRepository->findAll();
+        
         return $this->render('frontOffice/offre_stages/recrutement.html.twig', [
-            'controller_name' => 'OffreStagesController',
+            'offres' => $liste,
+        ]);
+    }
+    #[Route('/AfficheOffresFront', name: 'AfficheOffres')]
+    public function AfficheOffres(OffreStageRepository $stageRepository): Response
+    {
+        $liste = $stageRepository->findAll();
+        
+        return $this->render('frontOffice/offre_stages/recrutement.html.twig', [
+            'offres' => $liste,
         ]);
     }
+    
     
     
     
